php/mixphp-workerman/src/Vega.php

```diff
@@ -14,6 +14,7 @@ class Vega
     public static function new(): Engine
     {
         $vega = new Engine();
+        $vega->mode(Engine::FAST_MODE);
```

Comment:
what is **fast mode** @onanying ?

Reply:
@waghanza In fast mode, the user $request object is allowed to directly use Swoole or Workerman native request object to obtain parameters, which has better performance

Reply:
No other changes